airflow/www/templates/airflow/dags.html

```diff
@@ -81,9 +92,17 @@ <h2>DAGs</h2>
 
                 <!-- Column 3: Name -->
                 <td>
-                    <a href="{{ url_for('Airflow.'+ dag.get_default_view(), dag_id=dag.dag_id) }}" title="{{ dag.description }}">
-                        {{ dag.dag_id }}
-                    </a>
+                    <div style="float: left;">
```

Comment:
Do we need to have both float - isn't just floating the tags to the right enough?

Reply:
No, because the dag_id is in block display, so it will break the line after (but I changed it to span so the float is removed).